WebIDL/ecmascript-binding/class-string-iterator-prototype-object.any.js

```diff
@@ -31,16 +31,13 @@ test(t => {
 // was no @@toStringTag, it would fall back to a magic class string. This tests that the bug is
 // fixed.
 
-test(t => {
-  const proto = Object.getPrototypeOf(iteratorProto);
-  t.add_cleanup(() => {
-    Object.setPrototypeOf(iteratorProto, proto);
-  });
-
-  Object.setPrototypeOf(iteratorProto, null);
+test(() => {
+  const iterator = (new URLSearchParams()).keys();
+  assert_equals(Object.prototype.toString.call(iterator), "[object URLSearchParams Iterator]");
 
-  assert_equals(Object.prototype.toString.call(iteratorProto), "[object Object]");
```

Comment:
`iteratorProto` has own `@@toStringTag`, so this assertion seems incorrect.
Chromium fails it: https://wpt.fyi/results/WebIDL/ecmascript-binding/class-string-iterator-prototype-object.any.html